languages/he/feed.json

```diff
@@ -0,0 +1,9 @@
+{
+    "all-users": "כל המשתמשים"
+}
+{    
+    "No-posts-found.": "לא נמצאו פוסטים."
+}
+{
+    "Are-you-following-anyone?": "אתה עוקב אחרי מישהו?"
+}
```

Comment:
language files need to have one object with multiple keys, for example
{
  "all-users": "כל המשתמשים",
  "no-posts-found.": "לא נמצאו פוסטים.",
  "are-you-following-anyone": "אתה עוקב אחרי מישהו?"
}

Reply:
I fixed
Thank you very much for your patience
I'm still new to GitaHab
Including translating plugins to NODEBB